### DISCUSSION TITLE
How to restore specific files with Restic

### DISCUSSION THREAD

**Discussion by jmgilman** [vmware-tanzu/velero#5860] [Community support Q&A] [answered]
I found [this issue](https://github.com/vmware-tanzu/velero/issues/1210), but since it's quite a few years old at this point, most of the instructions don't seem applicable anymore. What I'm looking to do is to extract the files from a restic snapshot. I created a backup like so:

```
velero create backup restic --default-volumes-to-restic
```

And now, I would like to use the `restic` CLI to restore the files from it locally on my computer. It appears that the `node-agent` has the CLI tool on it, however, I can't figure out how to get the repository ID. Any tips?

**Comment by shubham-pampattiwar** [comment, marked as answer]
Once you create a restic based backup, velero creates a resource called `PodVolumeBackup`. The `PodVolumeBackup` resource has a spec called `RepoIdentifier` this should be the restic repository identifier.

**Reply by jmgilman** [reply]
Thank you! Do you by chance know if the restic snapshot ID is stored anywhere as well?

**Reply by shubham-pampattiwar** [reply]
The status of the `PodVolumeBackup` CR will consist of a field called `snapshotID` which is the identifier of the pod volume.
